docs/samples/Microsoft.ML.Samples/Program.cs

```diff
@@ -6,7 +6,7 @@ internal static class Program
     {
         static void Main(string[] args)
         {
-            LdaTransformExample.LdaTransform();
```

Comment:
>LdaTransformExample [](start = 12, length = 19)

Why is this being replaced? Don't we need the LDA sample anymore?

Reply:
This is just there to invoke the particular example we want to run. The LDA sample still exists

---
In reply to: [239576064](https://github.com/dotnet/machinelearning/pull/1812#discussion_r239576064) [](ancestors = 239576064)